clippy.toml

```diff
@@ -0,0 +1,2 @@
+too-many-lines-threshold = 225
+enum-variant-size-threshold = 300
```

Comment:
We should really not enable this globally since it has quite negative perf impact on the hot path.

Reply:
Yep, i couldn't find the incantation to do that per method for allow with values. Per call would be better ...

Reply:
You found the incantation so learning from that and marking as resolved 💅